bears/general/CPDBear.py

```diff
@@ -102,6 +108,8 @@ def run(self, language: str, minimum_tokens: int=20,
 
         stdout_output, _ = run_shell_command(arguments)
 
+        os.unlink(temp_filepath)
```

Comment:
maybe use `os.remove`? Seems they are functionally identical...

Imo `os.remove` is better to understand, what do you think? :)